Add Support Bundle link to Alloy UI navbar (#3938)
grafana/alloy: docs/sources/troubleshoot/support_bundle.md

about a running {{< param "PRODUCT_NAME" >}} instance, and can be used as a baseline of information when trying
to debug an [issue][alloy-repo].

> You can generate a support bundle by clicking the **Support Bundle** link in the {{< param "PRODUCT_NAME" >}} web UI navbar, or by accessing the `/-/support` endpoint directly.

Reviewer: ```suggestion
To generate a support bundle click the **Support Bundle** link in the {{< param "PRODUCT_NAME" >}} web UI navbar, or use the `/-/support` endpointy.
```
